backend/compact-connect/lambdas/custom-resources/handlers/compact_config_uploader.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+import json
+from decimal import Decimal
+from datetime import date
+from aws_lambda_powertools.utilities.typing import LambdaContext
+
+from config import config, logger
+
+
+def on_event(event: dict, context: LambdaContext):  # pylint: disable=inconsistent-return-statements,unused-argument
+    """
+    CloudFormation event handler using the CDK provider framework.
+    See: https://docs.aws.amazon.com/cdk/api/v2/python/aws_cdk.custom_resources/README.html
+
+    This custom resource uploads all the compact and jurisdiction configuration files into the provider DynamoDB table,
+    The configuration files are defined in the 'compact-config' directory under the 'compact-connect' directory.
+
+    This custom resource is defined in the CDK app within the 'CompactConfigurationUpload' construct of the
+    persistent stack.
+
+    :param event: The lambda event with the compact configuration in a JSON formatted string.
+    :param context:
+    :return: None - no infrastructure resources are created
+    """
+    logger.info('Entering compact configuration uploader')
+    properties = event['ResourceProperties']
+    request_type = event['RequestType']
+    match request_type:
+        case 'Create' | 'Update':
+            return upload_configuration(properties)
+        case 'Delete':
+            return
```

Comment:
Yes, in the case of delete we do not remove any data from the table, this will be handled by the DB's removal policy. If we retain the data for the DB, we don't want to delete anything here. If the removal policy deletes everything, then there is no need for us to duplicate that effort here.

Reply:
I'll add a comment to the code to denote this intended behavior